src/lib/lru-cache.test.ts

```diff
@@ -90,11 +90,13 @@ type Command =
     }
   | {type: 'removeLRU'}
 
-const arbitraryCommand: jsc.Arbitrary<Command> = jsc.record({
-  type: jsc.elements(['has', 'get', 'getSize', 'insert', 'getOrInsert', 'removeLRU']),
-  key: jsc.elements(['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']),
-  value: jsc.nat,
-}) as any
+const arbitraryCommand = fc.record({
```

Comment:
There is a built-in way to deal with commands in fast-check. I could probably migrate the test to it.